Session/Shared/FullConversationCell.swift

```diff
@@ -647,10 +651,10 @@ public final class FullConversationCell: UITableViewCell, SwipeActionOptimisticC
         fontSize: CGFloat,
         textColor: UIColor
     ) -> NSAttributedString {
-        guard !content.isEmpty, content != "NOTE_TO_SELF".localized() else {
+        guard !content.isEmpty, content != "noteToSelf".localized() else {
             return NSMutableAttributedString(
                 string: (authorName != nil && authorName?.isEmpty != true ?
-                    "\(authorName ?? ""): \(content)" :
+                    "\(authorName ?? ""): \(content)" : // stringlint:disable
```

Comment:
I guess it's more that strings are now a multi-platform issue so we should confirm that any string manipulation code is implemented consistently across platforms (including where RTL/LTR logic is and isn't being used) - even though the string is simple, if it was in Crowdin we wouldn't need to manually verify the implementation is consistent because everyone would be using the same string and formatting